liger_iris_pipeline/tests/test_image2.py

```diff
@@ -1,38 +1,58 @@
-import json
-import pytest
+# Imports
+import liger_iris_pipeline
+liger_iris_pipeline.monkeypatch_jwst_datamodels()
 import numpy as np
-import warnings
-import os
-from astropy.utils import data
+from liger_iris_pipeline.tests.test_utils import get_data_from_url
+import json
 from jwst.associations import load_asn
 
-import iris_pipeline
-
-iris_pipeline.monkeypatch_jwst_datamodels()
 
-from iris_pipeline.tests.test_utils import get_data_from_url
+######################
```

Comment:
remove #####

Reply:
Removed in [a59419d](https://github.com/oirlab/liger_iris_pipeline/pull/75/commits/a59419d97bb5fd630ac81691bc70a653b1213359)